cog.c

```diff
@@ -379,6 +394,11 @@ on_create_view (CogShell *shell, void *user_data G_GNUC_UNUSED)
                                                       "backend", view_backend,
                                                       NULL);
 
+    if (s_options.filter) {
+        WebKitUserContentManager *manager = webkit_web_view_get_user_content_manager(web_view);
+        webkit_user_content_manager_add_filter(manager, s_options.filter);
```

Comment:
You could free this after adding it

Reply:
Good catch, I'll post an update later covering this 👌🏼